types/react-beautiful-dnd/index.d.ts

```diff
@@ -8,6 +8,8 @@
 
 import * as React from 'react';
 
+type Maybe<T> = T | null | undefined
```

Comment:
I just changed all the instances of :? T from the original flow code into : Maybe<T> in ts code.
But for all the other cases that were typed explicitly in flow and that I turned into optional to respect the documented API I left as ?: T. Not it is easier to make a distinction between those cases.